src/nnet/nnet-blstm-projected-streams.h

```diff
@@ -19,8 +19,8 @@
 
 
 
-#ifndef KALDI_NNET_NNET_BLSTM_PROJECTED_STREAMS_H_
-#define KALDI_NNET_NNET_BLSTM_PROJECTED_STREAMS_H_
+#ifndef KALDI_NNET_BLSTM_PROJECTED_STREAMS_H_
```

Comment:
NNET_NNET is not a mistake: the file is referred to as "nnet/nnet-blstm-projected-streams.h". Please revert these guard names.